benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -635,6 +638,7 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "assignedToJudge", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "assignedToMedicalMember", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "caseBundles", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRUD"},
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "historicalBundles", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRUD"},
```

Comment:
This line is not needed since the role derives from caseworker-sscs which has CRUD access